### DISCUSSION TITLE
Status of stable 1.5.x versions

### DISCUSSION THREAD

**Discussion by intgr** [swc-project/swc#8917] [General]
Around April 29, apparently all 1.5.x versions were yanked from NPM repostories with the message:

> Mac OS installation is broken

* https://www.npmjs.com/package/@swc/core/v/1.5.2
* https://www.npmjs.com/package/@swc/core/v/1.5.1
* https://www.npmjs.com/package/@swc/core/v/1.5.0

But I couldn't find any further information anywhere. What exactly is "broken" and how broken is it? Has this issue been resolved or is it still being worked on? Are there plans for a subsequent release already?

I think there should be a pinned issue where people can find out more information, track the progress and be notified once this is resolved.

PS: This repository is using the next level of annoying bots for GitHub issues.

**Comment by kdy1** [comment]
My bad. I'm in travel/OoO and I forgot publishing a new version after it's fixed. It's already fixed and related PR closed #8898

**Reply by kdy1** [reply]
Published a new version with https://github.com/swc-project/swc/commit/0a6dbee9b1a42562cb84ce308688e7ba9eb2f6f8

**Comment by kdy1** [comment]
About the bot, you should read descriptions of fields if you are filing an issue. I don't think bot is wrong

**Reply by intgr** [reply]
I did read it. And the link field was unnecessary for this issue report.
